src/msvc_helper-win32.cc

```diff
@@ -58,14 +58,19 @@ int CLWrapper::Run(const string& command, string* output) {
     Win32Fatal("SetHandleInformation");
 
   PROCESS_INFORMATION process_info = {};
-  STARTUPINFOA startup_info = {};
+  STARTUPINFO startup_info = {};
   startup_info.cb = sizeof(STARTUPINFOA);
   startup_info.hStdInput = nul;
   startup_info.hStdError = ::GetStdHandle(STD_ERROR_HANDLE);
   startup_info.hStdOutput = stdout_write;
   startup_info.dwFlags |= STARTF_USESTDHANDLES;
 
-  if (!CreateProcessA(NULL, (char*)command.c_str(), NULL, NULL,
+#ifdef _WIN32
+  wstring commands = Utf8ToWide(command);
+  if (!CreateProcess(NULL, (wchar_t*)commands.c_str(), NULL, NULL,
```

Comment:
Does create_process really require a non-const string? Wtf?

Regardless, use const_cast<>, not c-style cast.

Reply:
Yes, see https://devblogs.microsoft.com/oldnewthing/20090601-00/?p=18083

Reply:
Wow, horrible API design.

Then this *cannot* call c_str().

Instead, the appropriate thing to do is

(slight psuedocode)
```cpp
wchar_t buffer[MAX_PATH];
memcpy(commands.begin(), commands.end(), buffer);
CreateProcess(NULL, buffer,...);
```